controllers/articlesController.js

```diff
@@ -269,4 +276,39 @@ export default class PostController {
         .json({ message: "Article was NOT posted, Server error" });
     }
   }
+  /**
+   * @description This creates report an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+
+  static async reportArticle(req, res) {
+    try {
+      const { slug } = req.params;
+      const { id: userId } = req.user;
+      const { description } = req.body;
+      
+      if (!description || undefined) {
```

Comment:
if(!description) checks whether the value is `null || undefined`. Please remove that `undefined`